src/fetchers/copy-fetcher.js

```diff
@@ -5,11 +5,10 @@ import BaseFetcher from './base-fetcher.js';
 import * as fs from '../util/fs.js';
 
 export default class CopyFetcher extends BaseFetcher {
-  async _fetch(): Promise<FetchedOverride> {
-    await fs.copy(this.reference, this.dest, this.reporter);
```

Comment:
I don't see this copy anywhere else, just removed. Was it not needed? Is something else performing the copy now?

Reply:
Since "file" dependencies are already on the disk, they are now copied at the linking stage, directly from the source instead of from the cache folder.